tests/phpunit/tests/option/option.php

```diff
@@ -705,7 +706,7 @@ static function () use ( $default_value ) {
 			}
 		);
 
-		$this->assertTrue( update_option( $option, $default_value ), 'update_option() should have returned true.' );
+		$this->assertTrue( update_option( $option, 'new value' ), 'update_option() should have returned true.' );
```

Comment:
```suggestion
		$this->assertTrue( update_option( $option, false ), 'update_option() should have returned true.' );
```
Since the default value has been changed, let's try passing the original default value: `false`.

This will test that the filtered default value means the original one is now ignored.